pkg/cmd/pr/merge/merge.go

```diff
@@ -293,6 +322,11 @@ func (m *mergeContext) merge() error {
 		return err
 	}
 
+	if m.addToMergeQueue() {
+		_ = m.infof("Pull request #%d will be added to the merge queue for %s when ready\n", m.pr.Number, m.pr.BaseRefName)
```

Comment:
What does "when ready" mean in this context? If it's waiting for all requirements to be met, e.g. CI status or PR reviews, let's say "when all requirements are met" so that it's clearer

Reply:
this is for consistency with the web UI which says "Merge when ready"
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request-with-a-merge-queue#adding-a-pull-request-to-a-merge-queue

Reply:
From reading this link, it looks like "ready" means waiting for review approvals and CI status. In that case, let's use the same wording as elsewhere in CI, i.e. "when all requirements are met". This will improve consistency (with the CLI at least) and avoid the user having to guess what "ready" means. (The web UI has an additional description next to a button, so here in the CLI we could be descriptive too.)